compaction/compaction.cc

```diff
@@ -435,9 +438,7 @@ class formatted_sstables_list {
 };
 
 std::ostream& operator<<(std::ostream& os, const formatted_sstables_list& lst) {
-    os << "[";
-    os << boost::algorithm::join(lst._ssts, ",");
-    os << "]";
+    fmt::print(os, "[{}]", fmt::join(lst._ssts, ","));
```

Comment:
Guys, why don't you simply print in the loop instead of joining this into a single huge string?